internal/transform/operation.go

```diff
@@ -13,10 +14,12 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
+type Claimants []Claimant
```

Comment:
no reason, if we take out the type declaration, we'll just need to define the later variable as a slice before appending the `CreateClaimableBalance` details:
```
var claimants []Claimant
```
thoughts if this would be better?

Reply:
yup I think so 👍